docs/qs-backup-setup.md

```diff
@@ -82,81 +89,155 @@ You should obtain a layout similar to this one (it is fine if you only have one
+4. Copy the link starting with https://
+
+<img src="/content/images/backup-setup/gitsetup-clonehttps.png" data-origin="content/images/backup-setup/gitsetup-clonehttps.png" alt="">
+
+5. In the **command prompt**, type `git clone <https://github.com/......>`. Replace the text between **< >** with the link you copied from GitHub. 'git clone' will create a local copy of your github repository. 
+
+6. A popup will appear. Login with your GitHub account to authenticate.
+
+<img src="/content/images/backup-setup/gitsetup-login.png" data-origin="content/images/backup-setup/gitsetup-login.png" alt="">
+
+7. Your repository is now synchronized with your computer. Copy your SuperMemo collection into the new folder. Your folder should look similar to the example below (*.gitignore* might be missing):
 
 <img src="/content/images/backup-setup/git-local-collection-repository.png" data-origin="content/images/backup-setup/git-local-collection-repository.png" alt="">
 
-6. <a href="/content/data/sm-main-commit.bat" target="_blank" rel="noopener">Download this .bat file</a> and save it in your local repository folder (where your `.git` directory is located). It contains the following commands:
+8. <a href="/content/data/sm-main-commit.bat" target="_blank" rel="noopener">Download this .bat file</a> and save it in your local repository folder (where your `.git` directory is located). It contains the following commands:
 
 ```bat
 git add -A && git commit -m "Update"
 git push
 ```
 
-7. Run `sm-main-commit.bat` (double click). If all went well, your should be able to see your collection in your GitHub repository web page.
+9. Run `sm-main-commit.bat` (double click). If all went well, your should be able to see your collection in your GitHub repository web page.
 
 #### Pushing (*\"saving\"*) your work to GitHub
 
 !> Every time you finish using SuperMemo, make sure to run `sm-main-commit.bat`.
 
 That's all ! Your collection is synchronized online, congratulations !
 
+
+
+#### Optional: Pin commit script to taskbar
+
+To make running sm-main-commit.bat more convenient, you can add it to taskbar. Windows doesn't let you add .bat files to taskbar so instead you can do the following:
+
+1. Right click on sm-main-commit.bat and click [create shortcut ![](/content/images/backup-setup/shortcut-creation.png)](/content/images/backup-setup/shortcut-creation.png '@tooltip-preview').
+
+2. The new shortcut still can't be added to taskbar. As such, right click properties and [go to shortcuts menu ![](/content/images/backup-setup/shortcut-edit.png)](/content/images/backup-setup/shortcut-edit.png '@tooltip-preview'). Add ```cmd /c``` (```cmd \c``` will not work) to the front of [ the target script ![](/content/images/backup-setup/shortcut-target.png)](/content/images/backup-setup/shortcut-target.png '@tooltip-preview') and press apply:
```

Comment:
It would be best if `/content/images/backup-setup/shortcut-target.png` showed E:\SuperMemo-Collection` for the drive instead of `C:\SuperMemo-Collection`, since this is the drive we have been illustrating this guide with.